paasta_tools/kubernetes_tools.py

```diff
@@ -955,6 +955,11 @@ def get_hacheck_sidecar_container(
         else:
             readiness_probe = None
 
+        hacheck_registrations_env = V1EnvVar(
+            name="SERVICE_REGISTRATIONS",
+            value=" ".join(self.get_registrations()),
+        )
```

Comment:
could always make this a function so that further below we could just do 
```Python
env=self.get_kubernetes_environment() + self.get_hacheck_env()
```, but that's pretty nit-picky and only slightly more aestheically pleasing :p